### ISSUE TITLE
iGPU passthrough in Proxmox 7 - help on the README

### ISSUE THREAD

**Issue by CaldeiraG** [vivekmiyani/OSX_GVT-D#2]
Hello there,

I found this repository [via Reddit](https://www.reddit.com/r/homelab/comments/qg37an/comment/hi3tn2w/?utm_source=share&utm_medium=web2x&context=3) and I was wondering if this configuration would work on my system.

Basically I wanna passthrough a HD 530 to my current macOS Monterey VM via Proxmox, if I do it via GUI it boot loops. What I'm confused about is that you apparently need to patch OpenCore to get it working (I have v15). If I then boot the VM with the patched OpenCore image and the GPU attached, will it work or its use is not intended for that?

I've always seen that iGPU passthrough is difficult and this is the closest thing I've come across to maybe get it working. 

Thanks in advance for the help.

**Comment by vivekmiyani** [comment]
Hello @CaldeiraG, This guide completely rely on [this](https://github.com/patmagauran/i915ovmfPkg) project. So you might want to check out my [issue](https://github.com/patmagauran/i915ovmfPkg/issues/8), you may find something helpful.

> What I'm confused about is that you apparently need to patch OpenCore to get it working (I have v15). If I then boot the VM with the patched OpenCore image and the GPU attached, will it work or its use is not intended for that?

I haven't use proxmox, I was directly running MacOS from QEMU. To debug the issue, you can use ssh.
Well actually I am not aware of the OpenCore patches you talking about, And I was able to boot the Catalina VM without patched OpenCore.

Have you saw the logging of OpenCore or MacOS?

**Comment by CaldeiraG** [comment]
Alright, I see what you mean. To clarify, I use a standard opencore ISO on Proxmox and it works just fine. This repo says to basically rebuild opencore to get the passthrough patched. Is this correct?

Even so, I'm not sure if this could work in Proxmox, let me know what you think.

**Comment by vivekmiyani** [comment]
> This repo says to basically rebuild opencore to get the passthrough patched. Is this correct?

No, we are not patching anything here. I think you are confused at `./opencore-rebuild.sh` command? (If yes then this command just create `OpenCore.qcow2` to pass it to VM. That's it.

> Even so, I'm not sure if this could work in Proxmox, let me know what you think.

I think it will work. see this thread (which was created by me when I faced some chrome related issues). One of the guy in this thread was doing passthrough in Proxmox (Faced same issue as mine - And finally we both succeed).

**Comment by vivekmiyani** [comment]
@CaldeiraG Forgot to mention the thread link, here you go:
https://www.reddit.com/r/hackintosh/comments/kzz2j7/chrome_and_electron_based_apps_crashing_in_fully/
